dashboard/app/controllers/admin_search_controller.rb

```diff
@@ -79,4 +79,19 @@ def show_pilot
     user_ids =  SingleUserExperiment.where(name: @pilot_name).map(&:min_user_id)
     @emails = User.where(id: user_ids).pluck(:email)
   end
+
+  def add_to_pilot
+    email = params[:email]
+    pilot_name = params[:pilot_name]
+    user = User.find_by_email_or_hashed_email(email)
```

Comment:
please add this here, copied from above: `return head :bad_request unless Experiment::PILOT_EXPERIMENTS.include?(pilot_name)`